CMSgov/cms-carts-seds
# Revert MDCT-199: Dependabot Patch

# Description

This reverts commit 985fa3ad04e62c6d87d21b89882f9e686fa472b7, reversing
changes made to 536e3227f84fc8b7d421ffc7ddfc3ec65cb5959b.

Merging this update caused the master branch to fail with the following message:

```
> iridat@0.1.0 build /app
> react-scripts build

Creating an optimized production build...
Failed to compile.

./src/App.scss
Node Sass version 7.0.1 is incompatible with ^4.0.0 || ^5.0.0 || ^6.0.0.
```

## How to test

* Ensure the revert is correct
* Ensure the deployment does not fail after merging to master

## Dependencies 

N/A

# Get it done

N/A

## Code authors checklist

## Reviewers Checklist (Two different people)

## Assignee
